### ISSUE TITLE
Uploaded HRM data cannot be transferred from PersonalTrainer to Flow

### ISSUE THREAD

**Issue by Zordid** [1ka/HRMUploader#1]
Hi! Thanks a lot for your great tool HRMUploader. It's a shame Polar cannot deliver what you can... :-1: 
I have tons of old data in ProTrainer that waits to be sent to the Polar Flow - since last week, the transfer of sessions from PersonalTrainer to Flow is finally possible. I think that Polar programmers have spent years on that... _cough_
But now I got a problem: sessions uploaded with HRMUploader are refused to be transferred to Flow. I have just one clue: the "Product" mentioned in the PersonalTrainer "data" section states "NO PRODUCT" for all my uploaded files with your tool. Maybe they allow just some product's data to be transferred? Is it possible to "fake" that product? In my case "RS800 CX" would be the right product...
Thanks!

**Comment by 1ka** [comment]
Hi Zordid,

Do you mean the product is empty when you view it on the Polar site, like the attached picture?

If so, email me a copy of a HRM file and I'll investigate further. (polar@softwyer.com)

![2015-08-24_19h55_20](https://cloud.githubusercontent.com/assets/240916/9449492/c87ce432-4a9a-11e5-8c61-b3a626a755fc.png)

**Comment by Zordid** [comment]
Correct! I got HRM files from Polar RS800CX and this is the result after uploading:

![noproduct](https://cloud.githubusercontent.com/assets/4798822/9460855/24503196-4b0a-11e5-9d61-dedfcc1aec21.png)

I really hope that this is the reason why Polar won't transfer these sessions to Flow... and that you can help fixing this... :-)

I'll send you the file!

**Comment by 1ka** [comment]
Had a quick look at this and I think it's a setup problem on the Polar site. For some reason, the site ignores the model data when ride data is uploaded. It appears to take the model data from the currently selected product. 

If you change the selected product to be your device, then the 'No Product' message will change.

![2015-09-07_08h57_17](https://cloud.githubusercontent.com/assets/240916/9711871/c8c93ec8-553e-11e5-868e-d2371910c5ef.png)
